modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -557,9 +557,28 @@ definition {
 	@ignore = "Test Stub"
 	@priority = "5"
 	test CanSelectAssetCreatorAsNotificationRecipientType {
+		property test.name.skip.portal.instance = "ProcessBuilderKaleoDesignerReact#CanSelectAssetCreatorAsNotificationRecipientType";
 
-		// TODO LRQA-73848 CanSelectAssetCreatorAsNotificationRecipientType pending implementation
+		Workflow.openWorkflowListView();
+
+		Workflow.addNewDefinition(workflowDefinitionTitle = "New Workflow Definition");
+
+		ProcessBuilderKaleoDesignerReact.selectNode(nodeType = "start");
+
+		ProcessBuilderKaleoDesignerReact.viewAndFillNotification(
+			name = "New Notification",
+			newNotification = "true",
+			notificationTemplateValue = "New Notification Template",
+			notificationTypeOption = "email",
+			recipientTypeOption = "Asset Creator");
+
+		ProcessBuilderKaleoDesignerReact.backArrowOnSidebar();
+
+		ProcessBuilderKaleoDesignerReact.gotoNotification(notificationName = "New Notification");
 
+		ProcessBuilderKaleoDesignerReact.viewNotificationInfo(
+			notificationRecipientType = "Asset Creator",
+			selectorField = "Recipient Type");
```

Comment:
This selectorField variable it's not present at the macro, the Fields are already configured, why you have used this? If you have any questions about variables or macros please reach me on slack or in person. Thanks.